deployment/windows/jellyfin.nsi

```diff
@@ -0,0 +1,381 @@
+    VIProductVersion "${ver_1}.${ver_2}.${ver_3}.0" ; VIProductVersion format, should be X.X.X.X
+    VIFileVersion "${ver_1}.${ver_2}.${ver_3}.0" ; VIFileVersion format, should be X.X.X.X
+    VIAddVersionKey "ProductName" "Jellyfin Server"
+    VIAddVersionKey "FileVersion" "${ver_1}.${ver_2}.${ver_3}.0"
+	VIAddVersionKey "LegalCopyright" "Jellyfin, Free Software Media System"
+	VIAddVersionKey "FileDescription" "Jellyfin Server"  
+	
+;TODO, check defaults
+    InstallDir "$PROGRAMFILES\Jellyfin" ;Default installation folder
+    InstallDirRegKey HKLM "Software\Jellyfin" "InstallFolder" ;Read the registry for install folder,
+  
+    RequestExecutionLevel admin ; ask it upfront for service control, and installing in priv folders
+	
+    CRCCheck on ; make sure the installer wasn't corrupted while downloading
+	
+    !define MUI_ABORTWARNING ;Prompts user in case of aborting install
+
+; TODO: Replace with nice Jellyfin Icons
+	!define MUI_ICON "${NSISDIR}\Contrib\Graphics\Icons\nsis3-install.ico" ; Installer Icon
+	!define MUI_UNICON "${NSISDIR}\Contrib\Graphics\Icons\nsis3-uninstall.ico" ; Uninstaller Icon
+	
+	!define MUI_HEADERIMAGE
+	!define MUI_HEADERIMAGE_BITMAP "${NSISDIR}\Contrib\Graphics\Header\nsis3-branding.bmp"
+	!define MUI_WELCOMEFINISHPAGE_BITMAP "${NSISDIR}\Contrib\Graphics\Wizard\nsis3-branding.bmp"
+
+;--------------------------------
+;Pages
+
+; Welcome Page
+	!define MUI_WELCOMEPAGE_TEXT "The installer will ask for details to install Jellyfin Server.$\r$\n$\r$\n$\r$\n\
+	ADVANCED:$\r$\n\
+	The default service install uses Local System account and is sufficient for most users. $\r$\n$\r$\n\
+	You can choose to install using Network Service account under advanced options. This also affects where Jellyfin Server and Jellyfin data can be installed. The installer will NOT check this, you should know what you are doing.$\r$\n$\r$\n\
+	You can choose the folder for Jellyfin Metadata under advanced options based on your needs."
+	!insertmacro MUI_PAGE_WELCOME
+; License Page
+    !insertmacro MUI_PAGE_LICENSE "$%InstallLocation%\LICENSE" ; picking up generic GPL
+; Components Page
+	!define MUI_COMPONENTSPAGE_SMALLDESC 
+    !insertmacro MUI_PAGE_COMPONENTS
+    !define MUI_PAGE_CUSTOMFUNCTION_PRE HideInstallDirectoryPage ; Controls when to hide / show
+	!define MUI_DIRECTORYPAGE_TEXT_DESTINATION "Install folder" ; shows just above the folder selection dialog
+    !insertmacro MUI_PAGE_DIRECTORY
+
+; Metadata folder Page
+    !define MUI_PAGE_CUSTOMFUNCTION_PRE HideDataDirectoryPage ; Controls when to hide / show
+	!define MUI_PAGE_HEADER_SUBTEXT "Choose the folder in which to install the Jellyfin Server metadata."
+	!define MUI_DIRECTORYPAGE_TEXT_TOP "The installer will set the following folder for Jellyfin Server metadata. To install in a differenct folder, click Browse and select another folder. Please make sure the folder exists. Click Next to continue."
+	!define MUI_DIRECTORYPAGE_TEXT_DESTINATION "Metadata folder"
```

Comment:
Is this metadata or just the jellyfin data folder?

Reply:
Yeah, it's the jellyfin data folder. I thought calling it metadata would make it explicit. What's the usual convention in our communications and documents ?

Reply:
Well there is also the actual metadata folder in jellyfin itself, like the one with all the images and nfo files. If not configured those are place next to the media.

Reply:
Yea, it should just be the "data folder/directory", since it's where all the "jellyfin data" lives. It's not really "metadata" but the main data of the application.